Explore Viam (#3186)
viamrobotics/docs: docs/get-started/drive-rover.md

cost: "0"
---

> If you haven't read through [Learn Viam](/get-started/), we recommend you do so, as it will provide you with an overview of the Viam platform.

Reviewer: ```suggestion
If you haven't read through [Learn Viam](/get-started/), we recommend you do so, to get an overview of the Viam platform.
```

> The next few quickstarts, will guide you through the concepts you need to get started with Viam.

Reviewer: ```suggestion
These quickstarts will guide you through the concepts you need to get started with Viam.
```
since they're already on one of the quickstarts, and there aren't any more except for these

Author: ```suggestion
This is the first of a few quickstarts which will guide you through the concepts you need to know to get started with Viam.
```
